### PR TITLE
CNV#27694: OpenShift Virtualization 4.15 release notes

### PR DESCRIPTION
**Pull request by sjhala-ccs**
Version(s): 4.15

Issue: [CNV-27694](https://issues.redhat.com//browse/CNV-27694)

Link to docs preview: https://72003--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/release_notes/virt-4-15-release-notes

Additional information: Also addresses [CNV-36081](https://issues.redhat.com//browse/CNV-36081)

### CODE REVIEW

File: virt/release_notes/virt-4-15-release-notes.adoc [hunk 1]
```diff
@@ -172,7 +169,7 @@ Deprecated features are included in the current release and supported. However,
 
 Removed features are not supported in the current release.
 
-//CNV-23499: Carry over/repeat removed feature from version 4.12 and 4.13
+//CNV-23499: Carry over/repeat removed feature from 4.12
```
Comment (sjhala-ccs) [inline virt/release_notes/virt-4-15-release-notes.adoc:172]: Should we still keep this note in for 4.15?

File: virt/release_notes/virt-4-15-release-notes.adoc [hunk 2]
```diff
@@ -161,6 +157,7 @@ Deprecated features are included in the current release and supported. However,
 * The `copy-template`, `modify-vm-template`, and `create-vm-from-template` tasks are deprecated.
 
 //CNV-29048 Release note: Metrics name changes
+//Check if there is a 4.15 article that we can link to
```
Comment (sjhala-ccs) [inline virt/release_notes/virt-4-15-release-notes.adoc:160]: Can I please get a link to the article with 4.15 metric changes

**Review by tiraboschi — APPROVED** [review]

File: virt/release_notes/virt-4-15-release-notes.adoc [hunk 3]
```diff
@@ -165,6 +161,7 @@ Deprecated features are included in the current release and supported. However,
...
 * Many OpenShift Virtualization metrics have changed or will change in a future version. These changes could affect your custom dashboards. See link:https://access.redhat.com/articles/7028805[OpenShift Virtualization 4.14 metric changes] for details. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2179660[*BZ#2179660*])
```
Comment (sjhala-ccs) [inline virt/release_notes/virt-4-15-release-notes.adoc:165]: Do we need a similar note as above for 4.15 metric changes? If so, can I please get a link to the relevant article?

File: virt/release_notes/virt-4-15-release-notes.adoc [hunk 4]
```diff
@@ -176,7 +173,7 @@ Deprecated features are included in the current release and supported. However,
 
 Removed features are not supported in the current release.
 
-//CNV-23499: Carry over/repeat removed feature from version 4.12 and 4.13
+//CNV-23499: Carry over/repeat removed feature from 4.12
 //Check if this is still needed for 4.15
 * Support for the legacy HPP custom resource, and the associated storage class, has been removed for all new deployments. In {VirtProductName} {VirtVersion}, the HPP Operator uses the Kubernetes Container Storage Interface (CSI) driver to configure local storage. A legacy HPP custom resource is supported only if it had been installed on a previous version of {VirtProductName}.
```
Comment (sjhala-ccs) [inline virt/release_notes/virt-4-15-release-notes.adoc:178]: Should we still need to keep the above note for 4.15?

File: virt/release_notes/virt-4-15-release-notes.adoc (hunk 3 above)
Comment (sjhala-ccs) [reply virt/release_notes/virt-4-15-release-notes.adoc:174]: @kmajcher-rh @sradco Can you please provide your input?

File: virt/release_notes/virt-4-15-release-notes.adoc (hunk 4 above)
Comment (sjhala-ccs) [reply virt/release_notes/virt-4-15-release-notes.adoc:187]: @aglitke Any thoughts on still keeping this for 4.15?

**Review by kalexand-rh — APPROVED** [review]
This is ok to merge after peer review.

File: virt/release_notes/virt-4-15-release-notes.adoc (hunk 3 above)
Comment (sjhala-ccs) [reply virt/release_notes/virt-4-15-release-notes.adoc]: I have commented this note out for 4.15

File: virt/release_notes/virt-4-15-release-notes.adoc (hunk 4 above)
Comment (sjhala-ccs) [reply virt/release_notes/virt-4-15-release-notes.adoc:187]: Leaving this note as is

**Comment by ousleyp** [comment]
LGTM

**Pull request merged**